### PR TITLE
My pre-course files

### PR DESCRIPTION
**Pull request by seand52**

### CODE REVIEW

File: staff/sean-daryanani/pre-course/calculadora/main.js [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+    else if (currentOperand === '*') {
+        result = parseFloat(firstNumber)* parseFloat(secondNumber)
+        input.value = Math.round(result*100000)/100000;
+    }
+
+    else if (currentOperand === '/') {
+        result = parseFloat(firstNumber)/ parseFloat(secondNumber)
+        input.value = Math.round(result*100000)/100000;
+        if (result === Infinity || result === -Infinity || result === NaN) {
+            input.value = "Can't divide by 0!"
```
Comment (maryammdot) [inline staff/sean-daryanani/pre-course/calculadora/main.js:100]: It's really cool, but with the operation 0/0 it doesn't work.

File: staff/sean-daryanani/pre-course/calculadora/main.js [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+        plusMinus.style.visibility = 'hidden';
+        percent.style.visibility = 'hidden'
+    }
+    else if (currentOperand === '/') {
+        operators[1].style.visibility= 'hidden';
+        operators[2].style.visibility= 'hidden';
+        operators[3].style.visibility= 'hidden';
+        plusMinus.style.visibility = 'hidden';
+        percent.style.visibility = 'hidden'
+    }
```
Comment (aleixcam) [inline staff/sean-daryanani/pre-course/calculadora/main.js:76]: switch case is a better option here

File: staff/sean-daryanani/pre-course/calculadora/main.js [hunk 3]
```diff
@@ -0,0 +1,313 @@
...
+
+}
+
+function hideOperators() {
+    if (currentOperand === '+') {
+        operators[0].style.visibility= 'hidden';
+        operators[1].style.visibility= 'hidden';
+        operators[2].style.visibility= 'hidden';
+        plusMinus.style.visibility = 'hidden';
+        percent.style.visibility = 'hidden'
```
Comment (anasanjuan) [inline staff/sean-daryanani/pre-course/calculadora/main.js]: as far as I understand, the use of  semicolon(;) is optional... but I see a pattern here that confuses me: why do you always omit only the last semicolon?

Comment (seand52) [reply staff/sean-daryanani/pre-course/calculadora/main.js]: I prefer to put semicolons "por si acaso". I really don't know why I missed it out in the last one always... no logic behind that! Thanks

File: staff/sean-daryanani/pre-course/calculadora/main.js (hunk 2 above)
Comment (seand52) [reply staff/sean-daryanani/pre-course/calculadora/main.js:76]: Thanks for the tip. I've never used switch case before though! 
Edited my code to use switch case statements in this function

File: staff/sean-daryanani/pre-course/calculadora/main.js [hunk 4]
```diff
@@ -0,0 +1,313 @@
...
+    })
+
+    //Square root click even
+    squareRoot.addEventListener("click", function() {
+        removeBackground();
+        sqrt();
+    })
+
+}
+
```
Comment (gbaste) [inline staff/sean-daryanani/pre-course/calculadora/main.js:309]: It does not let me put the second number in negative!

File: staff/sean-daryanani/cv.html [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+                <td align="center">Exeter</td>
+                <td align="center">PGCE Secondary Mathematics</td>
+            </tr>
+            </table>
+    </section>
+
+    <hr>
+
+    <footer>
+        <p>Find me on <a href="#">Linkedin</a></p> or <a href="">Github</a>
```
Comment (aleixcam) [inline staff/sean-daryanani/cv.html]: the p tag ends too soon

File: staff/sean-daryanani/cv.html [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Sean Curriculum</title>
+</head>
+
+<body>
+    <header>
+        <h1>Sean Daryanani | Aspiring Web Developer</h1>
+        <address>
+            <p><strong>Calle</strong>: XXXXXXXXXXX</p>
```
Comment (manuelbarzi) [inline staff/sean-daryanani/cv.html]: `strong` element here it seems to be used to mark titles in bold, but that's NOT the correct purpose of `strong` element, please RTFM https://www.w3schools.com/tags/tag_strong.asp

File: staff/sean-daryanani/cv.html [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+        <h2>Projects</h2>
+        <ul>
+            <li>Pasapalabra Game - This game is was made using HTML, CSS, and Javascript</li>
+            <li>Calculator - Web-based calculator that uses HTML, CSS, and Javascript</li>
+        </ul>
+    </section>
+
+    <section>
+        <h2>Work Experience</h2>
+        <table>
```
Comment (manuelbarzi) [inline staff/sean-daryanani/cv.html]: you seem to be using `table` element as a layout solution, but not as the correct purpose of this element, which is just for listing data in table format. please, if that's the case, correct it. and RTFM https://www.w3schools.com/html/html_tables.asp

File: staff/sean-daryanani/cv.html [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+                <td align="center">08/2014 to 08/2015</td>
+                <td align="center">Poole High School</td>
+                <td align="center">Poole</td>
+                <td align="center">Teacher of Mathematicsr</td>
+            </tr>
+        </table>
+    </section>
+
+    <section>
+        <h2>Education</h2>
```
Comment (manuelbarzi) [inline staff/sean-daryanani/cv.html]: same here

File: staff/sean-daryanani/pre-course/calculadora/main.js (hunk 3 above)
Comment (seand52) [reply staff/sean-daryanani/pre-course/calculadora/main.js]: I added a semicolon to all the lines now

File: staff/sean-daryanani/pre-course/calculadora/main.js (hunk 4 above)
Comment (seand52) [reply staff/sean-daryanani/pre-course/calculadora/main.js:309]: Thanks, fixed the bug!

File: staff/sean-daryanani/pre-course/calculadora/main.js (hunk 1 above)
Comment (seand52) [reply staff/sean-daryanani/pre-course/calculadora/main.js:100]: I tried fixing the bug but can't figure out why it's not working. My if condition includes result===NaN! Maybe you can help me tomorrow

**Pull request merged**